CMakeLists.txt

```diff
@@ -2178,7 +2194,7 @@ if(QT6)
     URI Mixxx.Controls
     VERSION 0.1
     RESOURCE_PREFIX /mixxx.org/imports
-    OPTIONAL_IMPORTS Mixxx
+    IMPORTS Mixxx
```

Comment:
This looks wrong, not every type in that module imports `Mixxx`. Also, why is this even part of this PR? It's not necessary to fix Mixxx on Ubuntu, right?

Reply:
It was necessary at one point during debugging. Now it still works even if I ommit the line entirely so it can be reverted.